tests/cli/demultiplex/test_create_sample_sheet.py

```diff
@@ -62,7 +62,9 @@ def test_create_bcl2fastq_sample_sheet(
     mocker,
 ):
     # GIVEN a flowcell directory with some run parameters
-    flowcell: FlowCellDirectoryData = FlowCellDirectoryData(flow_cell_working_directory)
+    flowcell: FlowCellDirectoryData = FlowCellDirectoryData(
+        flow_cell_working_directory, bcl_converter=BclConverter.BCL2FASTQ
```

Comment:
function with more than two parameters
```suggestion
        flow_cell_path=flow_cell_working_directory, bcl_converter=BclConverter.BCL2FASTQ
```